src/Symfony/Component/HttpKernel/Config/FileLocator.php

```diff
@@ -65,20 +65,30 @@ public function locate($file, $currentPath = null, $first = true)
             || (\strlen($file) > 3 && ctype_alpha($file[0]) && ':' === $file[1] && ('\\' === $file[2] || '/' === $file[2]))
             || null !== parse_url($file, PHP_URL_SCHEME)
         )) {
+            $triggerDeprecation = false;
+
             // no need to trigger deprecations when the loaded file is given as absolute path
             foreach ($this->paths as $deprecatedPath) {
                 if (\is_array($locations)) {
                     foreach ($locations as $location) {
```

Comment:
what about writing this as `foreach ((array) $locations as $location) {` and remove the is_array check just before?